ndk-sys/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Add `tool_type` getter for `Pointer` events. (#323)
```

Comment:
This needs to go in `ndk/CHANGELOG.md`, not `ndk-sys/CHANGELOG.md` :grimacing:

Reply:
Oh whoopsie